sdkproject/Assets/Mapbox/Unity/Editor/PropertyDrawers/CoreVectorLayerPropertiesDrawer.cs

```diff
@@ -118,17 +118,14 @@ private void DrawLayerName(SerializedProperty property, Rect position, List<stri
 				return;
 			}
 
-			if (!_isLayerNameGUIContentSet)
+
+			_layerTypeContent = new GUIContent[layerDisplayNames.Count];
+			for (int extIdx = 0; extIdx < layerDisplayNames.Count; extIdx++)
```

Comment:
Can we have a more descriptive variable name than "extIdx"? "exitIndex?" (That's what I assume it's short for)